### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Destiny-2048**
Minor changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Solar2D Game Engine
-## rebranded Corona CoronaSDK or Corona
+## Rebranded Corona SDK or Corona
```
Comment (Veter51) [inline README.md:2]: Maybe Corona SDK with a space would be better

Comment (Destiny-2048) [inline README.md:2]: Maybe Corona SDK with a space would be better

**Comment by sekodev** [comment]
I think just "Rebranded Corona SDK" is far better. IMO, repeating the same name is not entirely necessary.

(By the way, it could be a good idea to sign CLA to submit your changes.)

File: README.md (hunk 1 above)
Comment (Shchvova) [reply README.md:2]: Thanks. I agree

**Pull request merged**